lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -57,10 +59,10 @@ def serializable_hash_for_collection(serializer, options)
           hash
         end
 
-        def serializable_hash_for_single_resource(serializer, options)
+        def serializable_hash_for_single_resource(options)
           primary_data = primary_data_for(serializer, options)
           relationships = relationships_for(serializer)
-          included = included_for(serializer)
+          included = included_resources(@include_tree)
```

Comment:
I know there's a trend towards avoiding instance variables, but I think they make the situation really clear, compared to private accessors.